app/views/notes/_notes.html.erb

```diff
@@ -59,8 +59,8 @@
       </div>
 	    <style>
 		    .content {
-    width: 100px;
-    margin-bottom: 30%;
+ width: 100px;
+ margin-bottom: 30%;
```

Comment:
Please add closing bracket `}`. As you see bracket starting at line 61 is not closed.